src/LiveDevelopment/LiveDevMultiBrowser.js

```diff
@@ -807,6 +807,7 @@ define(function (require, exports, module) {
      */
     function setLauncher(launcher) {
         if (!(launcher && launcher.launch)) {
+            // Useful for debugging on a condition where launcher failed
```

Comment:
Let's just revert all the changes to this file, since none of them are really needed.